modules/server_new/src/main/scala/observe/server/keywords/StandardHeader.scala

```diff
@@ -26,9 +26,8 @@ final case class StateKeywordsReader[F[_]: Applicative](
   observer:   Option[Observer]
 ) {
 
-  def observerName: F[String]       =
-    observer.map(_.value).filter(_.nonEmpty).getOrElse("observer").pure[F]
-  def operatorName: F[String]       = operator.map(_.value).filter(_.nonEmpty).getOrElse("ssa").pure[F]
+  def observerName: F[String]       = observer.map(_.value.value).getOrElse("observer").pure[F]
+  def operatorName: F[String]       = operator.map(_.value.value).getOrElse("ssa").pure[F]
```

Comment:
Maybe `sso` or `operator`

Reply:
observer is initialized to sso in the UI. Should we set the operator too?

Reply:
nope, we don't really know who the operator is

Reply:
The observer should be set to the name of the logged in user

Reply:
Yup, that's done in the UI. Then it can be edited, but not left blank.